src/tests/unit/tests/common/message-creators/content-panel-action-message-creator.test.ts

```diff
@@ -15,88 +16,105 @@ import {
         };
-        expect(messagesPosted).toEqual([expectedMessage]);
+
+        actionMessageDispatcherMock.verify(dispatcher => dispatcher.dispatchMessage(It.isValue(message)), Times.once());
     });
 
-    it('creates closeContentPanel', () => {
-        const telemetry = Mock.ofType<BaseTelemetryData>().object;
-        telemetryDataFactoryMock.setup(tdf => tdf.fromDetailsViewNoTriggeredBy()).returns(() => telemetry);
+    it('dispatches for openContentHyperLink', () => {
+        const telemetry = { triggeredBy, source, href };
+        telemetryDataFactoryMock.setup(tdf => tdf.withTriggeredByAndSource(event, source)).returns(() => telemetry);
 
-        creator.closeContentPanel();
+        testSubject.openContentHyperLink(event, href);
 
-        const expectedMessage: Message = {
+        const message: Message = {
             payload: {
+                eventName: CONTENT_HYPERLINK_OPENED,
                 telemetry,
             },
-            tabId,
-            messageType: Messages.ContentPanel.ClosePanel,
+            messageType: Messages.Telemetry.Send,
         };
-        expect(messagesPosted).toEqual([expectedMessage]);
+        actionMessageDispatcherMock.verify(dispatcher => dispatcher.dispatchMessage(It.isValue(message)), Times.once());
     });
 
-    it('creates openContentPage', () => {
-        const telemetry = { triggeredBy, source, contentPath };
+    it('dispatches for openContentPanel', () => {
+        const telemetry = { triggeredBy, source };
         telemetryDataFactoryMock.setup(tdf => tdf.withTriggeredByAndSource(event, source)).returns(() => telemetry);
 
-        creator.openContentPage(event, contentPath);
+        testSubject.openContentPanel(event, contentPath);
 
-        const expectedMessage = {
+        const message: Message = {
             payload: {
-                eventName: CONTENT_PAGE_OPENED,
+                contentPath,
                 telemetry,
             },
-            tabId,
-            messageType: Messages.Telemetry.Send,
+            messageType: Messages.ContentPanel.OpenPanel,
+        };
+
+        actionMessageDispatcherMock.verify(dispatcher => dispatcher.dispatchMessage(It.isValue(message)), Times.once());
+    });
+
+    it('creates closeContentPanel', () => {
+        const telemetry = Mock.ofType<BaseTelemetryData>().object;
+        telemetryDataFactoryMock.setup(tdf => tdf.fromDetailsViewNoTriggeredBy()).returns(() => telemetry);
+
+        testSubject.closeContentPanel();
+
+        const message: Message = {
+            payload: {
+                telemetry,
+            },
+            messageType: Messages.ContentPanel.ClosePanel,
         };
-        expect(messagesPosted).toEqual([expectedMessage]);
+
+        actionMessageDispatcherMock.verify(dispatcher => dispatcher.dispatchMessage(It.isValue(message)), Times.once());
     });
 
-    it('creates openContentHyperLink', () => {
+    it('opense external links through the initiators', () => {
```

Comment:
Typo: opense should be opens
```suggestion
    it('opens external links through the initiators', () => {
```